doc/source/tracking/icepyx_pubs.bib

```diff
@@ -1,4 +1,23 @@
 % #########  materials about icepyx ################
+@misc{quest2023agu,
+author       = {Scheick, Jessica and
+                Bisson, Kelsey and
+                Fair, Zachary and
+                Piunno, Romina and
+                Abib, Nicole and
+                {Di Bella}, Alessandro and
+                Tilling, Rachel},
+title        = {On a {QUEST (Query, Unify, Explore SpatioTemporal)} 
+                  to Accelerate {ICESat-2} Applications in Ocean
+                  Science via {icepyx}},
+month        = jan,
+year         = 2024,
+publisher    = {Zenodo},
+version      = {v1},
```

Comment:
I added the Jupyter Notebook to the Zenodo, so I recommend updating this to v1.1.

Updated DOI for v1.1: 10.5281/zenodo.10563003